src/mn_setup.c

```diff
@@ -1074,7 +1091,9 @@ static void DrawInstructions()
         {
             if (menu_input == pad_mode)
             {
-                s = "[ PadA ] to toggle";
+                M_snprintf(pad_buf, sizeof(pad_buf), "[ %s ] to toggle",
+                           str_pad[STR_PADA]);
+                s = pad_buf;
```

Comment:
Is `M_snprintf` the best way to go here? I'm always wary of strings in C.

Reply:
Yes, when writing a formatted string to a fixed size buffer, I'd say `M_snprintf()` is the preferred method.